### PR TITLE
added !cocktail command

### LINKED ISSUE #54: cocktail lookup command

### PR DESCRIPTION
**Pull request by naterchrdsn**
resolves #54

### CODE REVIEW

File: commands/cocktail.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		var url = 'http://www.thecocktaildb.com/api/json/v1/1/search.php?s=';
+		if (!suffix) {
+			msg.channel.send('How about asking for something in specific?');
+		}
+		url += encodeURIComponent(suffix);
+		require('request')(url, function (err, res, body) {
+			var response = JSON.parse(body);
+			if (typeof response !== 'undefined' && response.drinks !== null) {
+				let result = response.drinks[0];
+        let ingredients = Object.entries(result).slice(7,22).map(entry => entry[1]);
```
Comment (develogix) [inline commands/cocktail.js]: indentation... TABS vs. SPACES

File: commands/cocktail.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+									name: 'Glass:',
+									value: result.strGlass,
+									inline: true
+							});
+					};
+					if (typeof ingredients !== 'undefined' && ingredients.length) {
+							let ingredientsList = '';
+							ingredients.forEach((element, index) => {
+									if (element !== '' && element !== '\n') {
+											if (ratios[index] !== '' && ratios[index] !== '\n') {
```
Comment (develogix) [inline commands/cocktail.js]: More indentation

File: commands/cocktail.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+													ingredientsList += `* ${element} - ${ratios[index]}\n`;
+											} else {
+													ingredientsList += `* ${element}\n`;
```
Comment (develogix) [inline commands/cocktail.js]: More indentation

File: commands/cocktail.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+											}
+									};
+							});
+							fields.push({
+									name: 'Ingredients:',
+									value: ingredientsList,
+									inline: true
```
Comment (develogix) [inline commands/cocktail.js]: More indentation

File: commands/cocktail.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+							});
+					};
+					if (typeof result.strInstructions !== 'undefined' && result.strInstructions !== '') {
+							fields.push({
+									name: 'Instructions:',
+									value: result.strInstructions,
```
Comment (develogix) [inline commands/cocktail.js]: More indentation

File: commands/cocktail.js [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+									title: `__${result.strDrink}__`,
+									author: {
+										name: 'CocktailDB',
+										url: 'http://www.thecocktaildb.com/',
+										icon_url: 'https://emojipedia-us.s3.amazonaws.com/thumbs/240/twitter/103/cocktail-glass_1f378.png'
+									},
+									thumbnail: {
+											url: thumbnail
+									},
+									fields: fields
```
Comment (develogix) [inline commands/cocktail.js]: More indentation

**Review by develogix — CHANGES_REQUESTED** [review]
Need to fix indentation. If you have VS Code setup to use tabs over spaces, you can just press `alt+shift+f`.

**Comment by naterchrdsn** [comment]
ready

**Review by develogix — APPROVED** [review]

**Pull request merged**